batch/batch/driver/instance_pool.py

```diff
@@ -110,6 +48,9 @@ def __init__(self, app, machine_name_prefix):
         self.max_instances = None
         self.pool_size = None
 
+        self.zones = ['us-central1-a', 'us-central1-b', 'us-central1-c', 'us-central1-f']
```

Comment:
This is the defaults until the zones get updated in the first iteration of the update_zones_loop?

Reply:
Yeah, exactly.  Hitting the /regions API is very slow (seconds), so I didn't want to do it synchronously in the startup code.  I'll add a comment to that effect.